### PR TITLE
plugin.video.thinktv 3.1.0

### PR DESCRIPTION
**Pull request by learningit**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.thinktv/resources/lib/scraper.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+          fanart = thumb
+          infoList ={}
+          infoList['Title'] = name
+          infoList['TVShowTitle'] = name
+          plot = b.get('description_long',b.get('description'))
+          infoList['Plot'] = h.unescape(plot)
+          infoList['Genre'] = b.get('genre_titles')
+          infoList['mediatype'] = 'tvshow'
+          c = (name, url, thumb, infoList)
+          contextMenu = [(self.addon.getLocalizedString(30009),'XBMC.RunPlugin(%s?mode=DF&url=AS%s)' % (sys.argv[0], str(c)))]
```
Comment (enen92) [inline plugin.video.thinktv/resources/lib/scraper.py:38]: please drop XBMC. prefix (was removed already in matrix)

File: plugin.video.thinktv/resources/lib/scraper.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+      maFile = xbmc.translatePath(os.path.join(profile, 'myshows.json'))
+      a = []
+      if os.path.exists(maFile):
+          with open(maFile) as infile:
+              a = json.load(infile)
+              for b in a:
+                  b = eval(b)
+                  name,url,thumb,infoList = b
+                  fanart = thumb
+                  contextMenu = [(self.addon.getLocalizedString(30010),'XBMC.Container.Update(%s?mode=DF&url=DS%s)' % (sys.argv[0], name))]
```
Comment (enen92) [inline plugin.video.thinktv/resources/lib/scraper.py:129]: same

File: plugin.video.thinktv/resources/lib/scraper.py (hunk 1 above)
Comment (learningit) [reply plugin.video.thinktv/resources/lib/scraper.py]: fixed

File: plugin.video.thinktv/resources/lib/scraper.py (hunk 2 above)
Comment (learningit) [reply plugin.video.thinktv/resources/lib/scraper.py]: fixed

**Pull request merged**